native/cocos/core/animation/SkeletalAnimationUtils.h

```diff
@@ -41,6 +42,24 @@ struct IJointTransform : RefCounted {
     IntrusivePtr<IJointTransform> parent;
 };
 
+class RealTimeJointTexture {
+public:
+    RealTimeJointTexture() {
+        textures.clear();
+    }
+
+    ~RealTimeJointTexture() {
+        textures.clear();
+        delete[] buffer;
+        buffer = nullptr;
+    }
+
+    static const uint32_t WIDTH = 256;
+    static const uint32_t HEIGHT = 3;
+    std::vector<IntrusivePtr<gfx::Texture>> textures;
+    float *buffer = nullptr;
```

Comment:
It is not a good way to public the member variables. If it just wants to record some data, than use struct is better.

Reply:
Can use `ccstd::vector<float>` instead?